ansible-hosts-file.tf

```diff
@@ -0,0 +1,21 @@
+resource "local_file" "hosts_ini" {
+  content = templatefile("${path.module}/hosts.ini.tpl",
+    {
+      broker_public_ips        = var.allocate_brokers_public_ip ? google_compute_instance.broker[*].network_interface.0.access_config.0.nat_ip : google_compute_instance.broker[*].network_interface.0.network_ip
+      broker_private_ips       = google_compute_instance.broker[*].network_interface.0.network_ip
+      client_public_ips          = google_compute_instance.client[*].network_interface.0.access_config.0.nat_ip
+      client_private_ips         = google_compute_instance.client[*].network_interface.0.network_ip
+      monitor_public_ip          = google_compute_instance.monitor[0].network_interface.0.access_config.0.nat_ip
+      monitor_private_ip         = google_compute_instance.monitor[0].network_interface.0.network_ip
+      ssh_user                   = var.ssh_user
+      enable_monitoring          = true
```

Comment:
[question] where is this used? shouldn't it be another var ?

Reply:
fixed, thought i caught all of those